Makefile

```
@@ -0,0 +1,15 @@
+all: test
```

Should we mark the targets as `.PHONY`?

```suggestion
.PHONY: all test install test coverage
all: test
```